crates/proof-of-sql-sdk/examples/produce-plan/args.rs

```diff
@@ -0,0 +1,38 @@
+use clap::Parser;
+use subxt::utils::H256;
+
+#[derive(Parser, Debug, Clone, PartialEq, Eq)]
+#[command(
+    name = "Proof of SQL Plan Producer",
+    version = "1.0",
+    about = "Produces the proof plan for a sql query."
+)]
+pub(super) struct ProducePlanArgs {
+    /// Url from which to retrieve the table schemas
+    #[arg(
+        long,
+        value_name = "SUBSTRATE_NODE_URL",
+        default_value = "wss://rpc.testnet.sxt.network",
+        env = "SUBSTRATE_NODE_URL"
+    )]
+    pub substrate_node_url: String,
+    /// SQL query to retrieve a plan for
+    #[arg(short, long, value_name = "QUERY", help = "SQL query to run")]
```

Comment:
I think the doc here is more accurate than the overridden help text, and clap will use the doc as the help text if we don't override it. Also I don't think the `value_name` is actually doing anything, though I know that's what the other CLI does.
```suggestion
    #[arg(short, long)]
```